fabric-chaincode-docker/README.md

```diff
@@ -0,0 +1,16 @@
+# Quick reference
+
+- **Maintained by**:  
+  [The Fabric Java chaincode maintainers](https://github.com/hyperledger/fabric-chaincode-java)
+
+# Overview
+
+This image is used by the default [Hyperledger Fabric](https://hyperledger-fabric.readthedocs.io/) chaincode builder when deploying Java smart contracts. It is not intended for use independently of Hyperledger Fabric.
+
+# Image variants
+
+Image tags correspond broadly to the major and minor version of Hyperledger Fabric for which they are primarily intended. More detailed information on image tags, interoperability, and supported Java versions can be found in the [compatibility](https://github.com/hyperledger/fabric-chaincode-java/blob/main/COMPATIBILITY.md) documentation.
```

Comment:
This was true in the past, but starting with Fabric v3 the Fabric and javaenv version numbers will deviate. But since it refers the reader to the compatibility doc for more details I think it's fine, I'll go ahead and merge.